easybuild/easyblocks/g/gromacs.py

```diff
@@ -182,6 +182,12 @@ def sanity_check_step(self):
         if self.toolchain.options.get('usempi', None):
             suff = '_mpi'
 
+        # Add the _d suffix to the suffix, in case of the double precission
+        if '-DGMX_DOUBLE=on' in self.cfg['configopts']:
+            suff = suff + '_d'
+        elif '-DGMX_DOUBLE=ON' in self.cfg['configopts']:
+            suff = suff + '_d'
```

Comment:
Sorry for the wrong hint. This is not working. The CMake cache still has the same value has on the command line.

The other approach that I just tested needs an addition CMake script. For example `cmake_test_variable.cmake`:
```
if(${VARNAME})
  message("${VARNAME} is ON")
else()
  message("${VARNAME} is OFF")
endif()
```
that you have to execute like this:
```
cmake -DVARNAME="GMX_DOUBLE" -P cmake_test_variable.cmake .
```